ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -13,12 +13,11 @@
 define([
     'underscore',
     'backbone',
-    'wellknown/wellknown',
     'usng.js/usng',
     'js/store',
     'js/Common',
-    'wreqr'
-], function (_, Backbone, wellknown, usngs, store, Common, wreqr) {
+    'require'
```

Comment:
✏️ Seems a bit odd to load require like this. If we need to use require, we could use it for all dependencies.

Reply:
Agreed, I will update this so the dependencies are loaded via `define` instead.